static/js/vue-cdr-access/src/components/full_record/bulkDownload.vue

```diff
@@ -35,17 +38,41 @@ export default {
             default: false,
             type: Boolean
         },
+        childCount: {
+            default: null,
+            type: Number
+        },
         workId: String
     },
 
     computed: {
+        hasTooManyChildren() {
+            // Based off of the limit set in DownloadBulkService
+            return this.childCount > 100;
+        },
+
         hasDownloadableContent() {
             return this.hasBulkDownloadAccess && this.totalDownloadSize !== null;
         },
 
         showTotalFilesize() {
             return this.hasBulkDownloadAccess && this.totalDownloadSize <= ONE_GIGABYTE;
         }
+    },
+
+    methods: {
+        handleDownload(event) {
+            if (this.hasTooManyChildren) {
+                // Prevent navigation if user cancels
+                if (!confirm("Number of files exceeds the download limit, only the first 100 will be exported, do you want continue?")) {
```

Comment:
This is a mouthful. Perhaps a period would flow better?
"Number of files exceeds the download limit, only the first 100 will be exported. Do you want to continue?"